include/aos/common/tools/fs.hpp

```diff
@@ -441,9 +441,12 @@ class FS {
             return Error(errno);
         }
 
+// Zephyr doesn't implement chmod
```

Comment:
Why do we need this chmod at all?

Reply:
to fix permissions. open creates file with masked permissions, which can be different from requested.